addons/mail/static/src/models/thread/thread.js

```diff
@@ -1509,23 +1510,23 @@ function factory(dependencies) {
         }
 
         /**
-         * Fetches followers of chats when they are displayed for the first
-         * time. This is necessary to clean the followers.
-         * @see `_onChangeFollowersPartner` for more information.
-         *
          * @private
          */
         _onChangeThreadViews() {
-            if (this.channel_type !== 'chat') {
-                return;
-            }
             if (this.threadViews.length === 0) {
                 return;
             }
-            if (this.areFollowersLoaded) {
-                return;
+            /**
+             * Fetches followers of chats when they are displayed for the first
+             * time. This is necessary to clean the followers.
+             * @see `_onChangeFollowersPartner` for more information.
+             */
+            if (this.channel_type === 'chat' && !this.areFollowersLoaded) {
+                this.refreshFollowers();
+            }
```

Comment:
IIRC it's the lazy clean of the problem we introduced with `@-`mention in chat, right?
If so, would be a great idea to make a task to remove this code in 6 months or so (we can schedule activity to do it).
What do you think @seb-odoo ?

Reply:
We can do it whenever we want in master but it needs to be together with a migration script. It's just not priority right now.

Reply:
Also note that if we ever merge members & follower which is likely to happen this year, that task will already take care of this problem.

Reply:
Ok, I was just suggesting planning removal in the future just so we don't keep weird stuffs indefinitely in code when we shouldn't need it. It's fine as long as we don't forget it.